internal/manifests/collector/ports.go

```diff
@@ -115,13 +115,28 @@ func getContainerPorts(logger logr.Logger, cfg string, specPorts []corev1.Servic
 }
 
 func getServicePortsFromCWAgentConfig(logger logr.Logger, config *adapters.CwaConfig) []corev1.ServicePort {
-	servicePortsMap := getAppSignalsServicePortsMap()
+	servicePortsMap := make(map[int32][]corev1.ServicePort)
+
+	fmt.Println("Inside get service ports")
+	if isAppSignalEnabled(config) {
+		fmt.Println("App Signals is enabled???")
+		addAppSignalServicePorts(servicePortsMap)
```

Comment:
can we follow the same pattern as below and have a function called `getApplicationSignalsReceiversServicePorts(logger, config, servicePortsMap)`

and then in that function we can make a call to `isAppSignalsEnabled(config)`